lib/plan_builder.rb

```diff
@@ -0,0 +1,90 @@
+class PlanBuilder
```

Comment:
Extract all data related methods to under the /data directory. That way, the methods here don't need to have a "name" as part of them.